service/src/main/java/uk/gov/hmcts/reform/fpl/events/CaseEvent.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+@Getter
+@EqualsAndHashCode
+public class CaseEvent {
```

Comment:
I don't think that this has been used, are all events supposed to extend it?

Reply:
you are right this one is orphaned, will remove